dev/com.ibm.ws.concurrent/src/com/ibm/ws/concurrent/internal/ManagedExecutorServiceImpl.java

```diff
@@ -454,6 +454,11 @@ public WSContextService getContextService() {
         return members;
     }
 
+    @Override
+    public PolicyExecutor getLongRunningPolicyExecutor() {
+        return longRunningPolicyExecutorRef.get();
```

Comment:
This will sometimes return NULL, per the JavaDoc,
```
* Null if longRunningPolicy is not configured, in which case the executor for the normal concurrency policy should be used instead.
```
The new interface on WSManagedExecutorService that you are adding needs to explicitly state what behavior you are getting in this regard.  Either document that the caller must expect to sometimes get a null and has its own responsibility for deciding what to do.  Or, document that the normal policy executor is returned in the absence of a long-running policy executor, in which case you would need to implement it this way.  The former would be more consistent with your current implementation.